setup.py

```diff
@@ -10,7 +10,20 @@
                     "docopt>=0.6",
                     "Jinja2>=2.7",
                     "PyYAML>=3.10",
-                    "python-dateutil>=2.2"]
+                    "python-dateutil>=2.2",
+                    # IPython dependencies:
```

Comment:
Because I do not need any surprises of unexpected updates.
Do you think I should not do that and lock only IPython dependency?